### PR TITLE
[ticket/14157] Allow to set the alt/title attribute on post icons

### PR DESCRIPTION
**Pull request by Zoddo**
[PHPBB3-14157](https://tracker.phpbb.com/browse/PHPBB3-14157)

### CODE REVIEW

**Comment by Zoddo** [comment]
Related to #3868

File: phpBB/adm/style/acp_icons.html [hunk 1]
```diff
@@ -89,6 +89,9 @@
 	<!-- ENDIF -->
 		<td>{L_WIDTH}</td>
 		<td>{L_HEIGHT}</td>
+	<!-- IF not S_SMILIES -->
+		<td>{L_TITLE}</td>
```
Comment (Nicofuma) [inline phpBB/adm/style/acp_icons.html]: Please use another language var, `Topic icons` doesn't tell us what it is

**Comment by Nicofuma** [comment]
looks good

File: phpBB/adm/style/acp_icons.html (hunk 1 above)
Comment (Zoddo) [inline phpBB/adm/style/acp_icons.html]: something like `alternative text`?

Comment (Nicofuma) [inline phpBB/adm/style/acp_icons.html]: should be better yes

Comment (Zoddo) [inline phpBB/adm/style/acp_icons.html]: Done

File: phpBB/phpbb/cache/service.php [hunk 2]
```diff
@@ -141,6 +141,7 @@ function obtain_icons()
 				$icons[$row['icons_id']]['img'] = $row['icons_url'];
 				$icons[$row['icons_id']]['width'] = (int) $row['icons_width'];
 				$icons[$row['icons_id']]['height'] = (int) $row['icons_height'];
+				$icons[$row['icons_id']]['alt'] = ($row['icons_alt']) ? $row['icons_alt'] : $row['icons_url'];
```
Comment (marc1706) [inline phpBB/phpbb/cache/service.php]: I'd rather have this empty than the icons URL. Nobody cares about the URL if there is no alternative text set.

Comment (Zoddo) [inline phpBB/phpbb/cache/service.php]: Done

**Pull request merged**